plugins/katello/nightly/advanced/http_proxies.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: plugins/katello/documentation
+title: Http Proxies
+version: 'nightly'
+---
+
+# Http Proxies
+
+## Http Proxy Support
+
+Katello enables external http proxies (provided by utilities such as [squid](http://www.squid-cache.org/) for repository operations such as synchronization.
+
+Http proxies can be created and then assigned to product though bulk selection as well as for each individual repository. Additionally, Katello provides http proxy policies for products or repositories. Policies include:
```

Comment:
s/Http proxies/HTTP proxies/
s/to product though/to a product through/
s/http proxy/HTTP proxy/